assets/optional/operator-lifecycle-manager/0000_50_olm_08-catalog-operator.deployment.x86_64.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: catalog-operator
+  namespace: openshift-operator-lifecycle-manager
+  labels:
+    app: catalog-operator
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    capability.openshift.io/name: "OperatorLifecycleManager"
+spec:
+  strategy:
+    type: RollingUpdate
+  replicas: 1
+  selector:
+    matchLabels:
+      app: catalog-operator
+  template:
+    metadata:
+      labels:
+        app: catalog-operator
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      serviceAccountName: olm-operator-serviceaccount
+      volumes:
+        - name: srv-cert
+          secret:
+            secretName: catalog-operator-serving-cert
+        - name: profile-collector-cert
+          secret:
+            secretName: pprof-cert
+      containers:
+        - name: catalog-operator
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: ["ALL"]
+          volumeMounts:
+            - name: srv-cert
+              mountPath: "/srv-cert"
+              readOnly: true
+            - name: profile-collector-cert
+              mountPath: "/profile-collector-cert"
+              readOnly: true
+          command:
+            - /bin/catalog
+          args:
+            - '--namespace'
+            - openshift-marketplace
+            - --configmapServerImage=quay.io/operator-framework/configmap-operator-registry:latest
```

Comment:
Is this right ref? `:latest`?

Reply:
That value should be replaced by `rebase.sh images`

Reply:
As comment above: I think PR should include results of `rebase.sh images`